lib/nxt_support/middleware/sentry_error_id.rb

```diff
@@ -7,7 +7,7 @@ def initialize(app)
 
       def call(env)
         status, headers, body = @app.call(env)
-        if status >= 500 && env['sentry.error_event_id']
+        if status && status.to_i >= 500 && env['sentry.error_event_id']
```

Comment:
I'm not sure if this is enough.
`"something".to_i == 0` which is not greater than 500.

Reply:
True, but if we have a middleware returning a non-numeric status code, we've got bigger problems. 🤣  Besides, that's not the purview of this middleware, and there are no negative effects either (header is only added if an error was reported).